Automattic/pocket-casts-ios
# Enables the Patron feature flag

Enables the Patron feature flag

## To test
1. CI is 🟢 
2. Launch the app
3. Sign into an account without Plus
4. Go to the upgrade screens and verify you see Patron listed
5. Go through the Purchase flow
6. ✅ Verify it works and Patron is unlocked

## Checklist
